src/components/camera/camera.tsx

```diff
@@ -9,14 +9,12 @@ declare var window: any;
 @Component({
   tag: 'pwa-camera',
   styleUrl: 'camera.css',
-  assetsDir: 'icons',
+  assetsDirs: ['icons'],
```

Comment:
`assetsDir` was removed in Stencil v3 and replaced with `assetsDirs` [(source)](https://stenciljs.com/docs/introduction/upgrading-to-stencil-three#deprecated-assetsdir-removed-from-component-decorator)